api/Cargo.toml

```diff
@@ -25,8 +25,10 @@ r2d2 = "0.8"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 uuid = "^0.7.0"
-validator = "0.10.0"
-validator_derive = "0.10.0"
+# validator = "0.10.0"
+# validator_derive = "0.10.0"
+validator = { git = "https://github.com/LucasPickering/validator", branch = "blanket-ref-impl" }
+validator_derive = { git = "https://github.com/LucasPickering/validator", branch = "blanket-ref-impl" }
```

Comment:
I have a PR up to this repo, once that gets merged and released we can go back to the stable version